### PR TITLE
Modal scroll fix

### PR DESCRIPTION
**Pull request by jayaike**
I made only the modal content scrollable while the buttons and the title remain fixed.

@pushkalkatara @vkartik97  PTAL

### CODE REVIEW

**Review by krtkvrm — CHANGES_REQUESTED** [review]
Hi @nsjcorps 
Can you please post the screenshot in which the changes are reflected

**Comment by jayaike** [comment]
How do I open the modal @vkartik97

**Comment by jayaike** [comment]
Screenshot (129)

- Screenshot of fix

**Comment by jayaike** [comment]
You can judge by the position of the scroll bar

**Comment by Sanji515** [comment]
Hey @nsjcorps there's one more modal for editing the phase details, which is not using the `utility/modal`. Can you please check this?

**Comment by jayaike** [comment]
@Sanji515 no problem. Can you take a look at the other pr on code splitting

**Comment by jayaike** [comment]
Screenshot 1_15_2020 8_51_36 PM


Please take a look @vkartik97 @Sanji515

**Review by Sanji515 — APPROVED** [review]
Thanks for the fix @nsjcorps , LGTM! This can be merged :tada:

**Comment by jayaike** [comment]
Thanks.

**Comment by jayaike** [comment]
Did you see the other PR? @Sanji515 https://github.com/Cloud-CV/EvalAI-ngx/pull/274

**Comment by Sanji515** [comment]
> Did you see the other PR? @Sanji515

You mean [this](https://github.com/Cloud-CV/EvalAI-ngx/pull/274) right? Not yet, looking to it now.

**Comment by jayaike** [comment]
Yes. That is correct. It's really just so you can see the significant difference if we lazy load @Sanji515

File: src/app/components/utility/modal/modal.component.html [hunk 1]
```diff
@@ -1,43 +1,47 @@
...
+              (change)="validateFileInput($event)" [isRequired]="item['isRequired']"
+              [label]="item['label']" [placeholder]="item['placeholder']"
+              [type]="item['type']" [icon]="item['icon']" [value]="item['value']"
+              #formmodal></app-input>
+            </div>
+          <small class="editor-validation-message">{{invalidFieldsAsText}}</small><br>
+        </div>
+      </div>
+      <div class="buttons">
+        <ul style="margin-right: 20px;" class="inline-list pointer">
```
Comment (lunayach) [inline src/app/components/utility/modal/modal.component.html]: Please move this style to `.scss` file.

Comment (jayaike) [reply src/app/components/utility/modal/modal.component.html]: @lunayach I have updated the code.

Comment (jayaike) [reply src/app/components/utility/modal/modal.component.html]: @Sanji515 I submitted my task btw :)

Comment (lunayach) [reply src/app/components/utility/modal/modal.component.html]: Thanks, @nsjcorps.

**Review by lunayach — APPROVED** [review]

**Pull request merged**